include/browseview_func.php

```diff
@@ -416,7 +416,7 @@ function getReversalEntries($letter = "", $page, $reversalLangcode = "", &$displ
 	$sortorderExists = (count($result))?TRUE:FALSE;
 
 	$alphabet = str_replace(",", "", get_option('reversal' . $reversalnr . '_alphabet'));
-	$collate = "COLLATE " . COLLATION . "_BIN";
+	$collate = "COLLATE " . MYSQL_CHARSET . "_BIN";
```

Comment:
The global constants `COLLATION` and `FULLCOLLATION ` were renamed to less confusing names, `MYSQL_CHARSET` ant `MYSQL_COLLATION`.